hm_pyhelper/hardware_definitions.py

```diff
@@ -1,30 +1,79 @@
 import os
+from enum import Enum, auto
+from collections import namedtuple
 from hm_pyhelper.exceptions import UnknownVariantException, \
-                                   UnknownVariantAttributeException
+    UnknownVariantAttributeException
 
+BALENA_ENV_RASPBERRY_PI_MODELS = [
+    'raspberry-pi2',
+    'raspberrypi3',
+    'raspberrypi3-64',
+    'raspberrypi4-64',
+    'nebra-hnt',
+    'raspberrypicm4-ioboard'
+]
 
-def is_rockpi():
-    return 'rockpi-4b-rk3399' in os.getenv('BALENA_DEVICE_TYPE')
+BALENA_ENV_ROCKPI_MODELS = ['rockpi-4b-rk3399']
 
+SBCInfo = namedtuple('SBCInfo', ['vendor_id', 'vendor_name', 'model_name'])
```

Comment:
Can you move the new code to it's own module/file?

Reply:
done

Reply:
Looks unchanged